docs/CHANGELOG.rst

```diff
@@ -51,6 +51,9 @@ Fixed
   BBDuk - STAR - featureCounts workflow
 - Make ``cuffnorm`` process attach correct expression data objects to
   samples
+- Pin ``sra-toolkit`` version to 2.9.0 in ``resolwebio/utils`` Docker image.
+  Before, "current" version was downloaded which caused that automated Docker
```

Comment:
Please, move the explanation of why this change was made to commit message's body and just note the change in the CHANGELOG without explanation.